boards/arm/frdm_k82f/frdm_k82f.yaml

```diff
@@ -14,6 +14,7 @@ supported:
   - gpio
   - i2c
   - nvs
+  - pit
```

Comment:
Replace with `counter`

Reply:
ok

Reply:
I don't think it's a problem if there are multiple counters. This is a list of supported interfaces, not specific IPs.

Reply:
ok